core/src/components/input/input.ionic.scss

```diff
@@ -11,6 +11,8 @@
   --border-color: #{$ionic-color-neutral-300};
   --highlight-color-valid: #{$ionic-color-success-400};
   --highlight-color-invalid: #{$ionic-color-error-400};
+  --placeholder-color: #{$ionic-color-neutral-500};
```

Comment:
Updated the value to match Figma.

Reply:
I'm waiting on confirmation on colors but it seems like 500 is the correct color. I vote for leaving it as is. We can create a new PR if it ends up being something else.

Reply:
Works for me 👍

Reply:
Confirmed that all fills will have the placeholder text with 500 so no changes are needed from us! Woot!